stable/stolon/values.yaml

```diff
@@ -34,13 +34,13 @@ serviceAccount:
 
 superuserUsername: "stolon"
 
-## password for the superuser (default is 40 random chars)
-# superuserPassword: ""
+## password for the superuser
+superuserPassword: "superuserPassword"
```

Comment:
Now you are setting default passwords. That's not good. If you don't want to auto-generate them, make them empty and use the `required` function. Provide an additional values.yaml for CI that sets passwords for CI only.